client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.html

```diff
@@ -262,7 +262,8 @@ <h2>{{ 'Motions' | translate }}</h2>
         </div>
 
         <mat-divider
-            *ngIf="categories.length || motionBlocks.length || hasAmendments() || perms.isAllowed('manage')"
+            *ngIf="(categories.length || motionBlocks.length || hasAmendments() || perms.isAllowed('manage'))
+            && (perms.isAllowed('manage') || selectedView === 'list' || operator.hasPerms('core.can_manage_config'))"
```

Comment:
Btw, this is really ugly and if more menu points are added, this gets increasingly more complex. I'd love an own component for the menu which only adds dividers if an element is displayed before and after it. But that's probably a bit overkill :D maybe something for the backlog (since we don't have enough things to do ^_^)

Reply:
~~could probably be solved using a structural directive. Might be complex~~
yes, backlog.